ngraph/core/reference/include/ngraph/runtime/reference/topk.hpp

```diff
@@ -133,35 +126,18 @@ namespace ngraph
                                     compare_min<T, U>);
                     }
                     // Write temp vector to output
-                    if (compute_max)
+                    switch (sort)
                     {
-                        switch (sort)
-                        {
-                        case op::v1::TopK::SortType::NONE: break;
-                        case op::v1::TopK::SortType::SORT_INDICES:
-                            std::sort(workspace.begin(),
-                                      workspace.begin() + k,
-                                      sort_indices_descending<T, U>);
-                            break;
-                        case op::v1::TopK::SortType::SORT_VALUES:
+                    case op::v1::TopK::SortType::NONE: break;
+                    case op::v1::TopK::SortType::SORT_INDICES:
+                        std::sort(
+                            workspace.begin(), workspace.begin() + k, sort_indices_ascending<T, U>);
+                        break;
+                    case op::v1::TopK::SortType::SORT_VALUES:
+                        if (compute_max)
```

Comment:
We normally use curly braces even if there's a single statement in the if. But if clang-format doesn't complain, you can leave it as is.